pkg/api/orchestrators.go

```diff
@@ -207,7 +225,7 @@ func getKubernetesAvailableUpgradeVersions(orchestratorVersion string, supported
 
 }
 
-func dcosInfo(csOrch *OrchestratorProfile, hasWindows bool) ([]*OrchestratorVersionProfile, error) {
+func dcosInfo(csOrch *OrchestratorProfile, hasWindows bool, onAzureStack bool) ([]*OrchestratorVersionProfile, error) {
```

Comment:
This is because kubernetesInfo, dcosInfo, swarmInfo, dockerceInfo are in the orchestratorsFunc, and they have to follow the structure of orchestratorsFunc (https://github.com/Azure/aks-engine/blob/master/pkg/api/orchestrators.go#L16). Similarly, the `hasWindows` variable is also not used in these functions, but have to exist.